internal/command/orgs/orgs.go

```diff
@@ -94,9 +94,13 @@ func slugFromArgOrSelect(ctx context.Context, orgSlug string, filters ...api.Org
 }
 
 func OrgFromFirstArgOrSelect(ctx context.Context, filters ...api.OrganizationFilter) (*api.Organization, error) {
```

Comment:
```suggestion
func OrgFromFirstArgOrSelect(ctx context.Context, filters ...api.OrganizationFilter) (*api.Organization, error) {
```
I feel like after my changes this function name is not honest anymore.

Reply:
Thought to rename it to `OrgFromEnvVarOrFirstArgOrSelect`